src/templates/ar/2019/base.html

```diff
@@ -3,79 +3,128 @@
+                      </a>
+                    </li>
+                    <li>
+                      <a href="https://github.com/HTTPArchive/almanac.httparchive.org" class="github">
+                        <img src="/static/images/github.png" height="78" width="80" alt="GitHub" loading="lazy" />
+                      </a>
+                    </li>
+                  </ul>
+                </li>
+                {% if supported_years | length > 1 %}
+                <li>
+                  {{ year_switcher() }}
+                </li>
+                {% endif %}
+                {% if supported_languages | length > 1%}
+                <li>
+                  {{ language_switcher() }
+                }</li>
+                {% endif %}
+              </ul>
+            </li>
+          </ul>
+        </nav>
       </div>
-      <button type="button" class="menu-btn" aria-label="Menu">
-        <img class="menu-icon" src="/static/images/menu.png" data-src-close="/static/images/close.png" alt="" height="21" width="27" />
-      </button>
-      <nav title="Mobile header menu" class="menu">
-        <a href="{{ url_for('table_of_contents', year=year, lang=lang) }}">Table of Contents</a>
-        <a href="{{ url_for('contributors', year=year, lang=lang) }}">Contributors</a>
-        <a href="{{ url_for('methodology', year=year, lang=lang) }}">Methodology</a>
-        <div class="misc">
-          <a href="https://httparchive.org/">
-            <img src="/static/images/ha-home.png" alt="HTTP Archive home" height="35" width="70">
-          </a>
-          <div class="social-media">
-            <a href="https://twitter.com/HTTPArchive" class="twitter">
-              <img src="/static/images/twitter.png" height="65" width="80" alt="Twitter" loading="lazy" />
-            </a>
-            <a href="https://github.com/HTTPArchive/almanac.httparchive.org" class="github">
-              <img src="/static/images/github.png" height="75" width="80" alt="GitHub" loading="lazy" />
-            </a>
-          </div>
-          {{ language_switcher() }}
-        </div>
-      </nav>
     </div>
   </header>
   {% endblock %}
 
-  {% block main %}{% endblock %}
+  <div class="container">
+    {% block main %}{% endblock %}
+  </div>
 
   {% block footer %}
     <footer class="alt-bg">
       <a class="navigation-logo home-logo" href="{{ url_for('home', year=year, lang=lang) }}"><img src="/static/images/logo.svg" alt="Web Almanac by HTTP Archive" height="39" width="202"></a>
-      <nav title="Footer menu" class="nav-items">
-        <a href="{{ url_for('table_of_contents', year=year, lang=lang) }}">Table of Contents</a>
-        <a href="{{ url_for('contributors', year=year, lang=lang) }}">Contributors</a>
-        <a href="{{ url_for('methodology', year=year, lang=lang) }}">Methodology</a>
-        {{ language_switcher() }}
+      <nav title="Footer navigation" class="nav-items">
```

Comment:
Do you know if there's any harm in having two nav sections on the page?

And if an AT user wants to jump to the first nav section, would it actually better for them to arrive at the footer, where there is no menu to open? 🤔

Reply:
> Do you know if there's any harm in having two nav sections on the page?

[Absolutely fine](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav#Usage_notes):

> A document may have several `<nav>` elements, for example, one for site navigation and one for intra-page navigation. `aria-labelledby` can be used in such case to promote accessibility.

> And if an AT user wants to jump to the first nav section, would it actually better for them to arrive at the footer, where there is no menu to open? 🤔

Nah I think this is fine. AT users should navigate to `main` content if they want the main content and `nav` if they want navigation.